api/tests.py

```diff
@@ -3,17 +3,18 @@
 from rest_framework import status
 from rest_framework.test import APIClient
 
-from apps.budget.choices import ExpenseGroupChoices, CategoryGroupChoices, UploadCategoryChoices
-from apps.budget.models import Budget, Expense, Category, CategoryMap
+from apps.budget.choices import ExpenseGroupChoices, CategoryGroupChoices, UploadCategoryChoices, RevenueGroupChoices
+from apps.budget.models import Budget, Expense, Category, CategoryMap, Revenue
 from apps.geo.models import Country
 
 
 class APITestCase(TestCase):
     def setUp(self):
         call_command('create_countries')
         self.country = Country.objects.first()
-        self.budget = Budget.objects.create(country=self.country, year=2021)
-        self.client = APIClient()
+        # self.budget = Budget.objects.create(country=self.country, year=2021)
```

Comment:
setUp() is called for **every test**. Changing setUp will affect all tests. Make sure these changes do not break other tests.